### DISCUSSION TITLE
City Hub broken from Pybricks

### DISCUSSION THREAD

**Discussion by SaharshSS** [pybricks/support#678] [Q&A]
My computer turned off during the Pybricks firmware installation process, and it will no longer turn on, even with fresh batteries. How do I fix it?

**Comment by dlech** [comment]
Are there no lights at all? If the light comes on purple, then just flash the firmware again.

**Reply by SaharshSS** [reply]
The light didn't turn on, but I took everything out (batteries and battery holder) and retried and the purple light came on. Thanks for the quick response!

**Reply by dlech** [reply]
Glad it is working again.
